frontend/src/router/RouterWrapper.tsx

```diff
@@ -31,6 +33,8 @@ export default function AppWrapper() {
             element={<UserProfile />}
           />
           <Route path={routes.SIGN_UP} element={<SignUp />} />
+          <Route path={routes.ASK_QUESTION} element={<CreatePost />} />
+          <Route path={routes.POST} element={<Post />} />
```

Comment:
```suggestion
          <Route path={routes.POST + "/:id"} element={<Post />} />
```
Ok so there is an endpoint for post, but what post?